contributing.md

```diff
@@ -81,6 +81,21 @@ This is to help as part of our effort to build an accessible toolkit (starting w
 * DO use verbs like GET.
 * DO NOT use verbs that are not already used like fetch.
 
+## <a name="accessibility"></a> Accessibility Guideline
+
+We'll follow this guideline to ensure the basic accessibility features for each control. 
+
+### Color & High Contrast themes
+* Controls must support the 4 high contrast themes by default on Windows, in addition to changing the theme while the app is running.
+* Controls must have a contrast ratio of at least 4.5:1 between the text (and images with text) and the background behind it.
+### Keyboard
+* Controls must support keyboard navigation (tabs and arrow keys), the tab order must be the same as the UI and non-interactive elements mustn't be focusable. 
```

Comment:
How do you want to handle this for WebView? We have OS-level bugs that prevent proper focus, etc.

Reply:
I assume once those OS-level bugs get fixed, the WebView would be compliant? I think it's ok as long as we know a fix is coming.